src/lib/metrics.js

```diff
@@ -0,0 +1,340 @@
+// Use of package.json for configuration
+import packageMeta from '../../package.json';
+
+import { makeLog } from './utils';
+import { selectors } from './store';
+import { colorsWithAlpha } from './constants';
+
+const log = makeLog('metrics');
+
+const GA_URL = 'https://ssl.google-analytics.com/collect';
```

Comment:
Any reason not to use [`testpilot-ga`](https://github.com/mozilla/testpilot-ga) here? It should be usable from websites (and is [in use by Send](https://github.com/mozilla/send/blob/f7dc86ab2b85aa745cd389b9f1a1b25d856f144e/app/metrics.js#L1)).

Reply:
Mainly because I haven't seen testpilot-ga until just now, I'll take a look